packages/server/src/sync/service.ts

```diff
@@ -115,6 +117,13 @@ export class SyncService extends Service {
 
         if (!_.isEqual(configWithoutEnabled, oldConduit.config)) {
           await this.conduits.updateConfig(oldConduit.id, configWithoutEnabled)
+        } else {
+          // updating the config will clear the number of errors.
+          // But if the config is identical we still want to clear it
+          const status = await this.status.fetch(oldConduit.id)
+          if ((status?.numberOfErrors || 0) > 0) {
```

Comment:
PP: Since those two conditions are equal

```suggestion
          if (status?.numberOfErrors) {
```